pkg/kubectl/polymorphichelpers/mapbasedselectorforobject_test.go

```diff
@@ -19,10 +19,10 @@ package polymorphichelpers
 import (
 	"testing"
 
+	api "k8s.io/api/core/v1"
+	extensions "k8s.io/api/extensions/v1beta1"
```

Comment:
Similarly `extensionsv1beta1`

Reply:
I know this will require bigger changes, but in the long run it will simplify figuring out what's still left when externalizing commands.